app/models/filters/criteria/filter_for_ca_homeless.rb

```diff
@@ -0,0 +1,15 @@
+class Filters::Criteria::FilterForCaHomeless < Filters::Criteria::Base
+  def applies? = input.coordinated_assessment_living_situation_homeless
+
+  # This needs to work correctly with project type filters, where it adds the
+  # potentially additional type of CA, but only if LivingSituation (3.917.1) is
+  # of a homeless type (6, 1, 18)
```

Comment:
Probably worth renaming this, CA was the previous name for CE.  Since we're re-writing a bit, can we make this:
```suggestion
class Filters::Criteria::FilterForCeHomeless < Filters::Criteria::Base
  def applies? = input.coordinated_assessment_living_situation_homeless

  # This needs to work correctly with project type filters, where it adds the
  # potentially additional type of CE, but only if LivingSituation (3.917.1) is
  # of a homeless type (6, 1, 18)
```